frontend/document/credocument.lua

```diff
@@ -58,6 +58,10 @@ local CreDocument = Document:new{
     default_css = "./data/cr3.css",
     provider = "crengine",
     provider_name = "Cool Reader Engine",
+
+    hide_nonlinear = false,
```

Comment:
`hide_nonlinear_flows` here too, to make it clearer it's related to "flows" elsewhere.